src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/FullDefenceResponseContentProvider.java

```diff
@@ -40,6 +41,10 @@
         content.put("responseTypeSelected", fullDefenceResponse.getDefenceType().getDescription());
         if (fullDefenceResponse.getDefenceType().equals(DefenceType.ALREADY_PAID)) {
             content.put("hasDefendantAlreadyPaid", true);
+            fullDefenceResponse.getPaymentDeclaration()
+                .flatMap(PaymentDeclaration::getPaidAmount)
+                .map(Formatting::formatMoney)
+                .ifPresent(amount -> content.put("amount", amount));
```

Comment:
`amount` is used in multiple place already (Claim amount & Statement of Means amount in bank). To keep it appropriate, can this be renamed as `paidAmount` or something more clear.